### PR TITLE
Backend/wip update endpoint

### PR DESCRIPTION
**Pull request by jafow**
# summary

adds update endpoint for wip_jurisdiction

### CODE REVIEW

File: backend/controllers/admin.js [hunk 1]
```diff
@@ -125,35 +132,268 @@ exports.publishWipJurisdiction = async (req, res, next) => {
...
+    editorUserId: userId,
+    updatedDataKeys: Object.keys(updatedWip),
+  })
+  const updater = updateWipJurisdictionChild(wipJurisdictionId)
+  const locationsUpdater = updateWipJurisdictionLocation(wipJurisdictionId)
+  let result = {}
+
+  try {
+    // wip_jurisdiction_url
+    const urls = await updater(req.db.WipJurisdictionUrl, updatedWip.urls)
```
Comment (jafow) [inline backend/controllers/admin.js:157]: general pattern is we pass the model and list of records to the `updater` and it does the delete + re-create flow

**Review by jafow — COMMENTED** [review]
:ok:

**Comment by jafow** [comment]
can run this against the local server with `make auth` in the `_misc/requests` folder
and then do `./update-wip-jurisdiction.sh`
from within the /admins folder

File: backend/controllers/admin.js (hunk 1 above)
Comment (jmensch1) [reply backend/controllers/admin.js:157]: nice

**Comment by jmensch1** [comment]
looks good! I'm wondering if we need to wrap everything in a transaction, otherwise errors might leave the DB in a weird state. I'm just gonna approve and play with it locally.

**Review by jmensch1 — APPROVED** [review]

**Pull request merged**